src/dsl/OpenApiBuilder.ts

```diff
@@ -31,7 +31,8 @@ export class OpenApiBuilder {
                 callbacks: {}
             },
             tags: [],
-            servers: []
+            servers: [],
+            webhooks: {}
```

Comment:
having this property initially is not compatible with OpenAPI 3.0.

Reply:
That's true. We will need to make it optional.

Reply:
Ah woops, I'll raise a fix